.idea/phpunit.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These weird files are still present. Im kinda scared what they will change in project. Is there any way to remove them or just ignore?

Reply:
The files are created because i use PhpStorm. The files are not harmful and should not cause a problem.